README.md

```diff
@@ -26,13 +26,15 @@ The linter parses the LookML files in the project and checks if there are any ex
   - This rule checks that each explore has an owner defined in its [tag parameter](https://cloud.google.com/looker/docs/reference/param-explore-tags)
     - `tags: ['owner:your_name'] `
 - FieldRequiresDescription
-  - This rule checks that all fields (measures, dimensions, and dimension_groups) have descriptions added.
+  - This rule checks that all non-hidden fields (measures, dimensions, and dimension_groups) have descriptions added.
 - FieldSqlHtmlRequiresUserAttributeWhenSearchTermsFound
   - This rule checks fields with the search term in the name use a specific user_attribute to limit the field access.
 - ViewWithDimensionsAndMeasuresHasOnePrimaryKey
   - This rule checks to make sure that views have one field defined as the primary key.
 - ViewWithManyFieldsRequiresFieldsHiddenByDefault
   - This rule checks that views with over 50 fields enumerated have the hidden_by_default parameter set.
+- ViewRequiresLabel
+  - This rule checks that all views have a user-friendly label defined.
```

Comment:
"user-friendly" is a bit meaningless here, since the rule simply checks for the existence of a label.